src/tests/countComment.test.js

```diff
@@ -0,0 +1,16 @@
+import getComment from '../module/countComment.js';
+
+describe('Test comment counter', () => {
```

Comment:
All your test are failing. Make sure that you have tests for all edge cases. Kindly think at least about the scenario in which there are 0 items on the page (counter function should return 0)

Reply:
hi @jerryowusu 
- how to fix it because it works from mine computer